packages/web/package-lock.json

```diff
@@ -2578,88 +2578,6 @@
       "resolved": "https://registry.npmjs.org/@ctrl/tinycolor/-/tinycolor-3.4.1.tgz",
       "integrity": "sha512-ej5oVy6lykXsvieQtqZxCOaLT+xD4+QNarq78cIYISHmZXshCvROLudpQN3lfL8G0NL7plMSSK+zlyvCaIJ4Iw=="
     },
-    "@cykura/sdk-core": {
```

Comment:
feels like too many lock changes

Reply:
yeah agreed. i rverted to main, removed node_modules, ran i, and this is what happenend. i think it's cause there were recently a lot of probably unwanted lock changes on a recent commit?

Reply:
ok yeah that makes sense. Might as well go back to goo